README.md

```diff
@@ -24,7 +24,7 @@ add new //modules// to the language, which adds functionality to the
 language, like [link label="links"](https://modmark.org), lists and more!
 ```
 
-The `[link]` syntax invokes a **module**, which lives outside the language itself, and the ability to use custom modules enhances the language and gives it capabilities way beyond what one single syntax set may. Make sure to check out the [playground](https://modmark.org) to see an example document which uses modules!
+The `[link]` syntax invokes a **module**, which lives outside the language itself, and the ability to use custom modules enhances the language and gives it capabilities way beyond what one single syntax set may. Make sure to check out the [playground](https://modmark.org/#&playground) to see an example document which uses modules!
```

Comment:
Is this the correct URL? It seems like a strange-looking URL to me, I would think it would be `https://modmark.org/playground`. But, I don't know much about react routing